pkg/app/piped/executor/ecs/ecs.go

```diff
@@ -157,6 +184,48 @@ func applyServiceDefinition(ctx context.Context, cli provider.Client, serviceDef
 	return service, nil
 }
 
+func applyStandaloneTask(
+	ctx context.Context,
+	cli provider.Client,
+	clusterDefinition types.Cluster,
+	awsVpcConfiguration types.AwsVpcConfiguration,
+	taskDefinition types.TaskDefinition,
+) (*ecs.RunTaskOutput, error) {
+	output, err := cli.RunTask(ctx, clusterDefinition, awsVpcConfiguration, taskDefinition)
+	if err != nil {
+		return nil, fmt.Errorf("unable to run ECS task %s: %v", *taskDefinition.TaskDefinitionArn, err)
+	}
+	return output, nil
+}
+
+func isStandaloneTask(in *config.ECSDeploymentInput) bool {
```

Comment:
I suppose it's better to implement this as a config.ECSDeploymentInput's method.

Reply:
thx, That's better 🙇🚀